packages/react/src/DefaultResourceTimeline/DefaultResourceTimeline.stories.tsx

```diff
@@ -1,31 +1,35 @@
 import { DiagnosticReport } from '@medplum/fhirtypes';
 import { useMedplum } from '@medplum/react-hooks';
-import { Meta } from '@storybook/react';
 import { useEffect, useState } from 'react';
 import { Document } from '../Document/Document';
 import { DefaultResourceTimeline } from './DefaultResourceTimeline';
+import { advanceTime, withMockedDate } from '../utils/storybook';
+import { Meta } from '@storybook/react';
+import { createIdGenerator } from '@medplum/core';
 
 export default {
   title: 'Medplum/DefaultResourceTimeline',
   component: DefaultResourceTimeline,
+  decorators: [withMockedDate],
 } as Meta;
 
 export const Basic = (): JSX.Element | null => {
   const medplum = useMedplum();
   const [resource, setResource] = useState<DiagnosticReport>();
+
   useEffect(() => {
+    const uuid = createIdGenerator();
     medplum
       .createResource<DiagnosticReport>({
+        id: uuid(),
```

Comment:
The `MemoryRepository` class (used inside `MockClient`) allows manually setting `id`, `meta.versionId`, and `meta.lastUpdated`.  I suspect that for most of the components, it's just about manually setting those values.